src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -601,7 +683,7 @@ __hypercall_resource_access_check(spdid_t dstid, spdid_t srcid, int capmgr_ignor
 }
 
 word_t
-hypercall_entry(word_t *ret2, word_t *ret3, int op, word_t arg3, word_t arg4)
+hypercall_entry(word_t *ret2, word_t *ret3, int op, word_t arg3, word_t arg4, word_t arg5)
```

Comment:
A separate PR for the fix to add back the extra argument would be great.  Try not and conflate too many functionalities/purposes into the same PR.